apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx

```diff
@@ -309,47 +309,40 @@ export const EditConnectionDetails: FunctionComponent<EditConnectionDetailsProps
-                                                                    } }
-                                                                />
-                                                            </Grid.Column>
-                                                            <Grid.Column width={ 6 } height={ 1 }>
-                                                                <p>Update connection password</p>
-                                                            </Grid.Column>                
-                                                        </Grid.Row>
-                                                    </Grid>
-                                                
-                                                    { isPasswordEditing && (
-                                                        <Field
-                                                            name={ property.name }
-                                                            className="addon-field-wrapper"
-                                                            type="password"
-                                                            key={ index }
-                                                            required={ true }
-                                                            label={ name }
-                                                            requiredErrorMessage={
-                                                                `${property.description.split("#")[ 0 ]} is  required`
-                                                            }
-                                                            showPassword={ t("common:showPassword") }
-                                                            hidePassword={ t("common:hidePassword") }
-                                                            placeholder={
-                                                                t("console:manage.features.userstores.forms." +
-                                                            "custom.placeholder",
-                                                                {
-                                                                    name: property.description.split("#")[ 0 ]
-                                                                })
-                                                            }
-                                                            data-testid={ 
-                                                                `${ testId }-form-password-input-${ property.name }` 
-                                                            }
+                                                    <Box display="flex" flexDirection="row">
+                                                        <Checkbox 
+                                                            sx={ { pb: 1.4, pl: 0,pr: 1, pt: 0 } }
```

Comment:
Is it okay to use sx prop ?

Reply:
It'd be okay I think. It has been already used in the codebase.

Reply:
Ideally, we should have a component-specific `.scss` if this is not meant to be reused.

Ref: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/authentication-flow-builder/components/predefined-flows-side-panel/predefined-flows-side-panel.scss

Reply:
Will fix this in a follow-up PR